### PR TITLE
Updated readme for dashboard: alex

### LINKED ISSUE #236: Write dashboard readme
As a developer, I want clear documentation that outlines the installation process, the structure of the pipeline, and how to run the app, so new contributors can easily get started and understand the functionality of the app.

Update the readme to include details on:
- Setup
- Installation

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #236 

Updated readme to include information on:
- Setup
- Installation
-  Usage

### CODE REVIEW

File: dashboard/README.md [hunk 1]
```diff
@@ -1,12 +1,74 @@
 # StarWatch Dashboard
-This folder contains the scripts to run and dockerise the dashboard for the StarWatch pipeline project. The pipeline was created using `streamlit` and queries data from a postgres database using `psycopg2`.
+This folder contains the scripts to run and dockerise the dashboard for the StarWatch pipeline project. The pipeline was created using `streamlit` and queries data from a postgres database using `psycopg2`. Along with this, requests are also made to the [NASA APOD API](https://api.nasa.gov/) and [ISS API](http://open-notify.org/Open-Notify-API/ISS-Location-Now/).
```
Comment (CalebMathers) [inline dashboard/README.md:2]: Good linking to APIs for people to check out if they so wish.

File: dashboard/README.md [hunk 2]
```diff
@@ -1,12 +1,74 @@
...
+bash deploy.sh
+```
+7. Now follow the terraform setup steps in the `../terraform` folder to provision an ECS task definition based off of this docker image.
 
 ## Usage
+- To use locally one can use the following command:
+```bash
+streamlit run pipeline.py
+```
+It is a good idea to ensure that the dashboard works with your credentials prior to provisioning the AWS architecture, it should be available via a local URL printed to terminal upon running the streamlit dashboard.
```
Comment (CalebMathers) [inline dashboard/README.md:59]: Good tip

File: dashboard/README.md [hunk 3]
```diff
@@ -1,12 +1,74 @@
...
+2. Create an ECR repository through terraform or the AWS UI.
+In order for your provisioned architecture one must dockerise their scripts and dependencies and push to an ECR repository.  
+For the next steps you will require AWS credentials and the ECR URI.
+
+3. Download the aws-cli:
+```bash
+brew install awscli
+```
+4. Verify your credentials:
+```bash
```
Comment (CalebMathers) [inline dashboard/README.md:38]: bash? more like bosh

**Review by CalebMathers — APPROVED** [review]

File: dashboard/README.md [hunk 4]
```diff
@@ -1,12 +1,74 @@
 # StarWatch Dashboard
```
Comment (MurasatoNaoya) [inline dashboard/README.md:1]: Good use of markdown for a clearer README. `

File: dashboard/README.md [hunk 5]
```diff
@@ -1,12 +1,74 @@
...
+7. Now follow the terraform setup steps in the `../terraform` folder to provision an ECS task definition based off of this docker image.
 
 ## Usage
+- To use locally one can use the following command:
+```bash
+streamlit run pipeline.py
+```
+It is a good idea to ensure that the dashboard works with your credentials prior to provisioning the AWS architecture, it should be available via a local URL printed to terminal upon running the streamlit dashboard.
 
+- To run tests run the following command:
```
Comment (MurasatoNaoya) [inline dashboard/README.md:61]: Clear instructions on running tests and confirming their coverage.

**Pull request merged**